server/admin.api.js

```diff
@@ -66,23 +70,9 @@ router.post('/admin/cleanup', mw.adminOnly, (req, res)=>{
 });
 
 /* Searches for matching edit or share id, also attempts to partial match */
-router.get('/admin/lookup/:id', mw.adminOnly, async (req, res, next)=>{
-	HomebrewModel.findOne({
-		$or : [
-			{ editId: { $regex: req.params.id, $options: 'i' } },
-			{ shareId: { $regex: req.params.id, $options: 'i' } },
-		]
-	}).exec()
-	.then((brew)=>{
-		if(!brew)	// No document found
-			return res.status(404).json({ error: 'Document not found' });
-		else
-			return res.json(brew);
-	})
-	.catch((err)=>{
-		console.error(err);
-		return res.status(500).json({ error: 'Internal Server Error' });
-	});
+router.get('/admin/lookup/:id', mw.adminOnly, asyncHandler(HomebrewAPI.getBrew('admin', false)), async (req, res, next)=>{
+	const brew = req?.brew ?? undefined;
```

Comment:
I don't think even this line is needed. If brew is undefined, `getBrew` has already failed and we wouldn't get this far.

Reply:
At this point, I think the whole function/endpoint is obsolete.

Reply:
Hm. Do we already have an endpoint that just gets a brew and returns it?

Reply:
Yes, getBrew in homebrew.api, but that returns the text and textbin as well. This function does not.

Reply:
`getBrew` is a middleware, not an endpoint.